config/history.go

```diff
@@ -137,6 +137,17 @@ var DefaultChains = map[common.ChainName]*History{
 	},
 	common.ChainNameTestnet: {
 		Records: []*Record{
+			{
+				// https://github.com/oasisprotocol/testnet-artifacts/releases/tag/2023-10-12
+				ArchiveName:   "2023-10-12",
+				GenesisHeight: 17751681,
+				ChainContext:  "0b91b8e4e44b2003a7c5e23ddadb5e14ef5345c0ebcb3ddcae07fa2f244cab76",
+				RuntimeStartRounds: map[common.Runtime]uint64{
+					common.RuntimeCipher:   1730319,
+					common.RuntimeEmerald:  2627790,
+					common.RuntimeSapphire: 2995927,
```

Comment:
I inferred these values by looking at the `genesis.json` under roothash -> runtime_states... see PR description

Reply:
Nit: PR description is empty. It's clear though that this is about https://github.com/oasisprotocol/testnet-artifacts/releases/download/2023-10-12/genesis.json, referenced from https://docs.oasis.io/node/testnet/